spec/models/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.create(Name: 'Tom', Photo: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Mexico.') }
+  before { subject.save }
+
+  it 'Name should not be blank' do
+    subject.Name = nil
+    expect(subject).to_not be_valid
+  end
+  it 'PostsCounter must be an integer ' do
+    subject.Posts_Counter = -1
+    expect(subject).to_not be_valid
+  end
+  it 'PostsCounter must be greater than or equal to zero. ' do
+    subject.Posts_Counter = 'word'
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] Kindly add a test for recent post method. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods)

Reply:
Hello, 
Please kindly check well my folder spec/models the first review didn't fit well all tests in different files, sorry for that, 
Happy R